axon-server-connector/src/test/java/org/axonframework/axonserver/connector/event/AxonServerStorageEngineBackedEventStoreIT.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.axonserver.connector.event;
+
+import io.axoniq.axonserver.connector.AxonServerConnection;
+import io.axoniq.axonserver.connector.AxonServerConnectionFactory;
+import io.axoniq.axonserver.connector.impl.ServerAddress;
+import org.axonframework.common.infra.MockComponentDescriptor;
+import org.axonframework.eventsourcing.eventstore.StorageEngineBackedEventStoreTestSuite;
+import org.axonframework.messaging.core.unitofwork.ProcessingLifecycle;
+import org.axonframework.messaging.eventhandling.conversion.EventConverter;
+import org.axonframework.test.server.AxonServerContainer;
+import org.axonframework.test.server.AxonServerContainerUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test suite implementation validating the {@link AxonServerEventStorageEngine}.
+ *
+ * @author John Hendrikx
+ */
+@Testcontainers
+class AxonServerStorageEngineBackedEventStoreIT extends StorageEngineBackedEventStoreTestSuite<AxonServerEventStorageEngine> {
+
+    private static final String CONTEXT = "default";
+
+    @SuppressWarnings("resource")
+    @Container
+    private static final AxonServerContainer container = new AxonServerContainer("docker.axoniq.io/axoniq/axonserver:2025.2.0").withDevMode(true);
```

Comment:
Assuming the branch is tailored to go into `main`, you can use the `withDcbContext(boolean)` method to make the context DCB-specific in one go.